docs/HowToIntegrate.md

```diff
@@ -1,41 +1,32 @@
-1. `npm install` all of the listed `dependencies` from the `package.json`
-1. If needed, modify or add templates to `./plugins/templateSelector`. The provided templates may need to be customized to correctly map data from the conversation payload to the populated Adaptive Card. To add new templates or modify existing templates, please refer to the [modifying templates] section.
-1. Configure your `templateSelector` plugin in `/plugins/templateSelector`:
-   - The `templateSelector` requires that for each `cardTemplateType` a coresponding template exists in (`templateSelector/templates`)
-   - Ensure your `/plugins/templateSelector/templateConfig.json` has the correct mappings of `cardTemplateType` to the template location.
-1. If needed, modify your conversation payload:
-   - If `cardTemplateType` does not exist on the conversation payload, add the field before passing the payload into the adapter.
-   - Note: Additional mappings on the conversation payload may be necessary to ensure a successful transformation in the adapter.
-1. Create a `pluginConfig.json` in your `/plugins` directory, and populate the config with the plugins and plugin location (template Selector is required, Pre and Post Processors are optional)
-1. Import your `pluginConfig.json` to your bot code (see below)
-1. Import the `adapter` to your bot code (see below)
-1. Import `cardFactory` from "botbuilder" into your bot code (see below)
+## Getting started with the Adaptive Card Transformer
+
+To use the `adaptive-card-transformer` you will need plugins, a `pluginConfig.json`, and a bot. To kickstart a project using the `adaptive-card-transformer`, leverage the Yeoman [generator-AdaptiveCardTransformer](https://github.com/retaildevcrews/generator-AdaptiveCardTransformer) to scaffold plugins, a `pluginConfig.json`, and an echo bot (if needed).
+
+## Automatic Integration
+
+1. If you do not have a bot, you can scaffold an echo bot with the [generator-AdaptiveCardTransformer](https://github.com/retaildevcrews/generator-AdaptiveCardTransformer).
+1. After scaffolding the echo bot and selected plugins, the generator will prompt the user to overwrite certain files. This will automatically integrate the `adaptive-card-transformer` and plugins into the scaffolded echo bot.
+1. Add logic and templates to your template-selector, pre-processor, and post-processor plugins. For an example, please refer to the [Adaptive Card Transformer Sample Bot](https://github.com/retaildevcrews/AdaptiveCardTransformerExampleBot/tree/main/src)
+1. Run `npm start` to run the echo bot with with the `adaptive-card-transformer`
+
+## Manual Integration
+
+If you already have a bot, follow these steps to integrate the `adaptive-card-transformer` into your bot:
+
+1. In your bot project, scaffold plugins and the `pluginConfig.json` with the Yeoman [generator-AdaptiveCardTransformer](https://github.com/retaildevcrews/generator-AdaptiveCardTransformer).
```

Comment:
Add step before `Add dependency on the `adaptive-card-transformer` to run cp .npmrc and add in token?